octane-eclipse-plugin/src/com/hpe/octane/ideplugins/eclipse/ui/comment/EntityCommentComposite.java

```diff
@@ -138,6 +138,8 @@ public void setEntityModel(EntityModel entityModel) {
     private void postComment(String text) {
         commentText.setEnabled(false);
         showControl(loadingComposite);
+        loadingComposite.setLocation(0, -300);
```

Comment:
I don't like this solution, it's random. Please modify the loading composite code, add a method that allows you to control where the loading gif is shown. ex: loadingComposite.setIndicatorLocation(IndictatorPosition.TOP)